test/utils.js

```diff
@@ -2,43 +2,10 @@
 
 const assert = chai.assert;
 
-export class AccuracyCriterion {
-  constructor(atol, rtol) {
-    this.atol = atol;
-    this.rtol = rtol;
-  }
-}
-
-export const opFp32AccuracyCriteria =
-    new AccuracyCriterion(1e-6, 5.0 * 1.1920928955078125e-7);
-
-// The following 2 constants were used for converted tests from NNAPI CTS
-export const ctsFp32RestrictAccuracyCriteria =
-    new AccuracyCriterion(1e-5, 5.0 * 1.1920928955078125e-7);
-export const ctsFp32RelaxedAccuracyCriteria =
-    new AccuracyCriterion(5.0 * 0.0009765625, 5.0 * 0.0009765625);
-
-// Refer to onnx/models
-//   https://github.com/onnx/models/blob/master/workflow_scripts/ort_test_dir_utils.py#L239
-// See details of modelFp32AccuracyCriteria setting:
-//   https://github.com/webmachinelearning/webnn-polyfill/issues/55
-export const modelFp32AccuracyCriteria = new AccuracyCriterion(1e-3, 1e-3);
-
-export function almostEqual(a, b, criteria) {
-  const delta = Math.abs(a - b);
-  if (delta <= criteria.atol + criteria.rtol * Math.abs(b)) {
-    return true;
-  } else {
-    console.warn(`a(${a}) b(${b}) delta(${delta})`);
-    return false;
-  }
-}
-
-export function checkValue(
-    tensor, expected, criteria = opFp32AccuracyCriteria) {
+export function checkValue(tensor, expected) {
   assert.isTrue(tensor.size === expected.length);
   for (let i = 0; i < expected.length; ++i) {
-    assert.isTrue(almostEqual(tensor.getValueByIndex(i), expected[i], criteria));
+    assert.equal(tensor.getValueByIndex(i), expected[i]);
```

Comment:
I suppose we should use `almostEqual` for float values comparison even the calculation is in double precision.

Could you use the ULP-based comparison and set the ULP tolerance to 0?

Reply:
@huningxin I think that the expected data here is indeed the finally output data which we could directly use it by WPT tests, that is,  we will directly convert these tests of this webnn-baseline repo to WPT. WDYT? Thanks.

Reply:
You could use these double precision expected data for WPT test as the baseline results. However, here these data are used to test the baseline implementation. You can image that these code may produce slightly different value on different JavaScript engine or architecture. So as I mentioned, if it compares float points, we should use `almostEqual` which could be ULP based.

Reply:
OK, I will update it by ULP based comparison. Thanks.

Reply:
@huningxin I've updated the commit, please take another look, thanks.